### ISSUE TITLE
Update Epsilon tools image

### ISSUE THREAD

**Issue by szschaler** [mdenet/educationplatform-docker#64]
@agarciadom has provided a new image for Epsilon tools, this should be linked from the docker-compose file: https://github.com/epsilonlabs/playground-backend/pkgs/container/playground-backend/316380500?tag=ep-tool-server

**Comment by szschaler** [comment]
I think https://github.com/mdenet/educationplatform-docker/blob/4cb0978b3dd6841bece1563eefe7dc7f8f3b41dd/docker-compose.yml#L46 currently pulls `playground-micronaut`, which strangely seems to still work and, at least on our dev server seemed to give us an updated diagram rendering. 

It pulls a version `0.1`, but the new `ep-tool-server` tag doesn't seem to come with a version number (or am I missing something?). Does this mean, we can only choose to always pull `latest` as it were?

**Comment by agarciadom** [comment]
Yes, we haven't deleted the old images to avoid breaking your setup, but they won't get updates anymore.

It's true that the new tags don't have version numbers: I wasn't sure we could just have arbitrary image names or if they had to match the name of the repository. I'll see if we can do something combining the variant, a Git tag in this repo, and the version of Epsilon being used (2.5). Would that work for you?

**Comment by szschaler** [comment]
I'm fine with whatever works for you. We have previously discussed people hosting their own tools on their own servers, so we could even look into that...

**Comment by agarciadom** [comment]
No worries - I can create a tag in the playground-backend repo and use that as a ref in a GHA matrix build. I'll give it a go and let you know how it goes.

**Comment by agarciadom** [comment]
Just adding to this that we reorganised how we tag Docker builds, so now you can have images that point to tagged versions of both the server and Epsilon itself, for stability. See here:

https://github.com/epsilonlabs/playground-backend/pkgs/container/playground-backend%2Fep-tool-server
